ex1/ex1.c

```diff
@@ -9,6 +9,19 @@
 int main(void)
 {
     // Your code here
+pid_t pid;
+int x;
+x = 12;
+printf("parent is %d value of x in the parent %d\n", pid, x);
 
+pid = fork();
+
```

Comment:
When you fork, remember to include some error handling in case something goes wrong, so the compiler/program can exit cleanly. There is an example of this in the README.